cfgov/jinja2/v1/landing-page-1/index.html

```diff
@@ -11,10 +12,42 @@
 {%- endblock %}
 
 {% block content_main %}
+    {{ hero.render() }}
     <div class="block
                 block__flush-top">
         {# TODO: Add main molecules and organisms. #}
-        {{ hero.render() }}
+
+        {# TODO: Add stacked image and text organism with bottom padding. #}
+        <div style="padding-bottom: 60px;">
+            <h2>Headline for 2/7 organism</h2>
+            {{ image_text_25_75.render({
+            'image_path': 'http://placehold.it/300x300',
+            'image_alt': '',
+            'heading' : 'Placeholder',
+            'body': 'Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.',
```

Comment:
Added in https://github.com/cfpb/cfgov-refresh/commit/8cd3c7c0377de184558b1543953032a2101bb97c